app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventPresenter.java

```diff
@@ -53,11 +58,100 @@ private boolean verify() {
         return true;
     }
 
+    private boolean findEmptyLink(String uri) {
+        if (uri == null)
+            return false;
+        else return !("".equals(uri));
+    }
```

Comment:
Remove this method, simply use `StringUtils.isEmpty`

Reply:
Or better even, create a new static method called `StringUtils.emptyToNull` which returns null if string is empty or string itself if it is not